### PR TITLE
Refactor/improveMapSegue

### PR DESCRIPTION
**Pull request by cordhernandez**
Disables Map Segue when Side Menu is open
Fixes reloading issue when Map View transitions to Table View
#102

### CODE REVIEW

File: BlackNectar/BlackNectar/GlobalLocationManager.swift [hunk 1]
```diff
@@ -113,13 +113,7 @@ class UserLocation: NSObject, CLLocationManagerDelegate, MKMapViewDelegate  {
                     .withPriority(.medium)
                     .send()
             
-            default:
-                
-                LOG.warn("User status not determines")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/GlobalLocationManager.swift:118]: How come the aroma message is removed from here?

File: BlackNectar/BlackNectar/StoresTableViewController.swift [hunk 2]
```diff
@@ -47,23 +42,21 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate,
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        stores.removeAll()
-        
         UserLocation.instance.initialize()
-        configureSlideMenu()
+        configureSideMenu()
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresTableViewController.swift:46]: Nice! Amazing what one character can do to enhance the meaning of a name.

File: BlackNectar/BlackNectar/StoresTableViewController.swift [hunk 3]
```diff
@@ -284,30 +271,95 @@ extension StoresTableViewController {
...
+        distanceFilter = UserPreferences.instance.distanceFilter
+        showStores = UserPreferences.instance.isStore
+        
+    }
+    
+}
+
+//MARK: SWReveal Delegate
+extension StoresTableViewController: SWRevealViewControllerDelegate {
+    func revealController(_ revealController: SWRevealViewController!, panGestureRecognizerShouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer!) -> Bool {
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresTableViewController.swift]: I think you can remove this extension and the stuff in it. I was experimenting with it.

File: BlackNectar/BlackNectar/StoresTableViewController.swift [hunk 4]
```diff
@@ -284,30 +271,95 @@ extension StoresTableViewController {
...
+//MARK: UI Screen Pan Gesture Code
 extension StoresTableViewController {
     
     func setEdgeGesture() {
         
-        edgeGesture.addTarget(self, action: #selector(self.handleRightEdge(gesture:)))
-        edgeGesture.edges = .right
-        edgeGesture.delegate = self
+        edgePanGestureRecognizer.addTarget(self, action: #selector(self.handleRightEdge(gesture:)))
+        edgePanGestureRecognizer.edges = .right
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresTableViewController.swift:339]: Amen!

**Review by SirWellington — APPROVED** [review]
:100: Looks good. Just remove that un-needed code and merge it in.

File: BlackNectar/BlackNectar/GlobalLocationManager.swift (hunk 1 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/GlobalLocationManager.swift:118]: done!

File: BlackNectar/BlackNectar/StoresTableViewController.swift (hunk 2 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/StoresTableViewController.swift:46]: thank you sir

File: BlackNectar/BlackNectar/StoresTableViewController.swift (hunk 3 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/StoresTableViewController.swift]: Mission Complete sir!

**Comment by SirWellington** [comment]
merge it like merge sort

File: BlackNectar/BlackNectar/StoresTableViewController.swift (hunk 4 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/StoresTableViewController.swift:339]: 🙏🏽

**Pull request merged**